libs/dsb-transport-nats-js/src/nats-jetstream-transport.ts

```diff
@@ -140,6 +163,48 @@ export class NATSJetstreamTransport implements ITransport {
         return res;
     }
 
+    public async subscribe(fqcn: string, clientId: string, cb: any): Promise<void> {
```

Comment:
this could be used for websocket right? is there anything else using it now?

Reply:
yes. it is just used in addressbook for now